app/helpers/export_helper.rb

```diff
@@ -26,8 +51,32 @@ def export_excel(project, components_type)
       component.rules.each do |rule|
         # fast_excel unfortunately does not provide a method to modify the @last_row_number class variable
         # so it needs to be manually kept track of
+        csv_attributes = rule.csv_attributes
+        if is_disa_export
+          check_text, fix_text = get_check_and_fix_text(rule.status)
```

Comment:
if above suggestion is accepted, this will also need to change
```suggestion
          check_text, fix_text = get_check_and_fix_text(rule.status).values_at('check_text', 'fix_text')
```